spec/system/audit_system_spec.rb

```diff
@@ -53,12 +53,13 @@
         expect(page).to have_content(item.name)
       end
 
-      it "hides the items quantity in the display" do
+      it "does not display quantities in line-item drop down selector" do
         create(:storage_location, :with_items, item: item, item_quantity: 10)
         visit subject
         first('.storage-location-source').all("option").last.select_option
         item_css = "#audit_line_items_attributes_0_item_id option[value='#{item.id}']"
-        item_text = find_all(item_css).first.text
+        item_text = find(item_css).text
+
```

Comment:
I'am not sure if this is still testing what we are seeking as described in the `it` statement. This test is only checking if an element exists but doesn't tell us the 'content' or 'text' included in there. Maybe a better way would be to test the content via:
```
find('option', text: 'Item Name', exact: true)
```

This would ensure that the item name appears and has to be exact (aka no quantities).

Let me know what you think.

Reply:
Ok cool thanks! I went ahead and made that changed and pushed it up.